source/simulator/stokes_matrix_free.cc

```diff
@@ -1282,12 +1356,6 @@ namespace aspect
     parse_parameters(prm);
     CitationInfo::add("mf");
 
-    if (sim.parameters.mesh_deformation_enabled)
```

Comment:
can you add an assert that disables mesh_deformation + elasticity for now?

Reply:
Like this?
`AssertThrow(!(sim.parameters.mesh_deformation_enabled && sim.parameters.enable_elasticity), ExcNotImplemented());`
then I will need to remove  free_surface_viscous_cylinder_2D_loading_gmg.prm

Reply:
See what @naliboff had to say on slack.